lib/fortify/base.rb

```diff
@@ -55,11 +55,11 @@ def cannot(action, *fields)
       access_list.delete(action) if access_list[action].empty?
     end
 
-    def initialize(record)
+    def initialize(object)
       raise InvalidUserError.new("Fortify user not set") if !Fortify.user
 
       @user = Fortify.user
-      @record = record
+      @record = object.is_a?(Class) ? object.new : object
```

Comment:
Not sure if there's a better way to do this. Any ideas?

Context: https://app.clubhouse.io/updater/story/3071/fortify-policy-record-support-broken

Reply:
I think this works, the only other thing would be `responds_to? :new`

Reply:
@justin-rhoades i like `responds_to? :new` more.. thanks!